### ISSUE TITLE
Inviting a user to a node - not working

### ISSUE THREAD

**Issue by robotnic** [buddycloud/buddycloud-server-java#308]
When I send this request
```
3:::{"type":0,"data":["xmpp.buddycloud.subscription",{"node":"/user/angular-xmpp@buddycloud.org/posts","jid":"robotnic@buddycloud.org","subscription":"invited"}],"id":30}
```
the user robotnic@buddycloud.org does not get an invite message.

**Comment by lloydwatkin** [comment]
If you retrieve subscriptions for the user do you see the invite in the list?

The functionality is certainly working, I wonder if @imaginator needs to update the version of `xmpp-ftw-buddycloud` deployed?

**Comment by robotnic** [comment]
Yes, it's in the list

subscriptions:
[{"subscription":"invited","jid":{"domain":"buddycloud.org","user":"robotnic"},"node":"/user/burma@buddycloud.org/posts"},...]

affiliations:
{"node":"/user/burma@buddycloud.org/posts","affiliation":"none","jid":{"domain":"buddycloud.org","user":"robotnic"}}

**Comment by lloydwatkin** [comment]
Ok so invites are working. Were you connected and had presence sent from
the invited user? Its something I've been working with so "know" it  works.
Again I'd potentially put it down to version issues. Have you tried to run
your own xmpp-ftw setup locally to confirm?

_____________________________________________________

Steven Lloyd Watkin
Software Engineer
PHP ::: Java ::: Ruby ::: Node.js ::: XMPP :: + many more
lloyd@evilprofessor.co.uk (email+jid) ::: http://www.evilprofessor.co.uk
Github / Twitter / Flickr / Facebook: lloydwatkin

Organiser of WORLD RECORD breaking charity event:
Scuba Santas ::: http://www.scuba-santas.co.uk
Supporting the RNLI & DDRC - 14th December 2014 - NDAC, Chepstow

**Comment by robotnic** [comment]
I install the newest version of xmpp-ftw-demo. 

The user robotnic@buddycloud.org invites rrr@buddycloud.org to the node
/user/robotnic@buddycloud.org/posts
```
3:::{"type":0,"data":["xmpp.buddycloud.subscription",{"node":"/user/robotnic@buddycloud.org/posts","jid":"rrr@buddycloud.org","subscription":"invited"}],"id":16}
```
xmpp-ftw debug:

info: Stanza received: 
```
<message type="headline" to="robotnic@buddycloud.org/786e0229-28fc-4218-86bc-67896c6484ac" from="channels.buddycloud.org" xmlns:stream="http://etherx.jabber.org/streams">
<event xmlns="http://jabber.org/protocol/pubsub#event">
<subscription invited-by="robotnic@buddycloud.org" jid="rrr@buddycloud.org" subscription="invited" node="/user/robotnic@buddycloud.org/posts"/>
</event>
</message>
```

I think the message should got to rrr instead to robotnic.

If I use the XML console and send the message to the CORRECT user it works.

**Comment by lloydwatkin** [comment]
I agree in this case the push should be going to rrr@buddycloud.org and robotnic@buddycloud.org should just be getting a result stanza. This now makes things a bug in the server and I'll take a look when I get a chance. Thanks!
